projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -890,4 +899,21 @@ private function is_user_connected() {
 	private function get_site_id() {
 		return Connection_Manager::get_site_id();
 	}
+
+	/**
+	 * Check if the posts are synced.
+	 *
+	 * @return bool True if is sync, false otherwise.
+	 */
+	private function is_posts_synced(): bool {
```

Comment:
Not a blocker :)
```suggestion
	private function is_full_sync_of_posts_in_progress(): bool {
```

Reply:
The initial sync, before the one triggered after 10 minutes, doesn't include the full post sync, so I don't want to show an empty post list until I'm sure that the `jetpack_full_sync_posts` happened, this is why I'm checking if it is sync instead of is in progress, relying on the check of `$full_sync_module->get_status()['config']['posts']`.

Reply:
Yes, that makes sense to me 👍 Maybe also consider `are_posts_ready` instead?